test/integration/RouteProxyTest.js

```diff
@@ -46,10 +49,31 @@ describe('Route proxy', () => {
   it('should support registering full URLs manually', done => {
     mockyeah.get('/http://localhost:8888/foo?ok=yes', { text: 'bar', status: 500 });
 
-    async.series([
-      cb => supertest(proxiedApp).get('/foo').expect(200, cb),
-      cb => request.get('/http://localhost:8888/foo?ok=yes').expect(500, 'bar', cb)
-    ], done);
+    async.series(
+      [
+        cb =>
+          supertest(proxiedApp)
+            .get('/foo')
+            .expect(200, cb),
+        cb => request.get('/http://localhost:8888/foo?ok=yes').expect(500, 'bar', cb)
+      ],
+      done
+    );
+  });
+
+  it('should support registering full URLs manually without leading slash', done => {
+    mockyeah.get('http://localhost:8888/foo?ok=yes', { text: 'bar', status: 500 });
+
+    async.series(
+      [
+        cb =>
+          supertest(proxiedApp)
+            .get('/foo')
+            .expect(200, cb),
+        cb => request.get('/http://localhost:8888/foo?ok=yes').expect(500, 'bar', cb)
+      ],
+      done
+    );
```

Comment:
We could look for opportunities to swap out all of this usage of `async` with `async/await` and `Promise.all` for parts that require concurrency.